cmd/common-main.go

```diff
@@ -51,23 +51,34 @@ func initConfig() {
 }
 
 func handleCommonCmdArgs(ctx *cli.Context) {
-	// Set configuration directory.
-	{
```

Comment:
IMO you don't need to remove the comment and the block pattern.  This will reduce diff

Reply:
I can add comments if you think it is helpful. I think the difference is small enough that it is OK to remove the block pattern.